analyzer/queries.go

```diff
@@ -25,7 +25,7 @@ func (qf QueryFactory) LatestBlockQuery() string {
 func (qf QueryFactory) IsGenesisProcessedQuery() string {
 	return `
 		SELECT EXISTS (
-			SELECT 1 FROM processed_geneses
+			SELECT 1 FROM multichain.processed_geneses
```

Comment:
why is this outside of oasis_3?

Reply:
Because contents are chain-independent. See also the comment at `CREATE DOMAIN`.
Discussed in Slack: We can't be 100% certain that we won't want to change any of the `multichain` tables once we actually have multiple chains, but it's a fair guess, and easy to undo/fix later.